### ISSUE TITLE
Unable to build "Release" version

### ISSUE THREAD

**Issue by feoff3** [DymOK93/KTL#307]
Error "warning treated as error" when trying to build Release version. 
Warning: Unreachable code at modules\fmt\core.hpp lines 624, 2212

`cmake --build . --config=Release`

CMake version 3.22.2

**Comment by DymOK93** [comment]
Yes, this is a known issue with the older version of [fmtlib](https://github.com/fmtlib/fmt) currently being used (see #144)
